packages/mail-client/src/mail-client.ts

```diff
@@ -283,13 +271,15 @@ function teardown(transport: Transporter<SentMessageInfo>, socket?: Socket) {
  * This function also does the destination look up, when passing `DestinationOrFetchOptions`.
  * @param destination - A destination or a destination name and a JWT.
  * @param mailConfigs - A single object or an array of {@link MailConfig}.
+ * @param smtpTransportOptions - A {@link SmtpTransportOptions} that defines how to set up an SMTP transport, including SSL and tls configurations.
  * @param mailClientOptions - A {@link MailClientOptions} that defines the configurations of the mail client, e.g., whether the mails are sent in parallel.
  * @returns A promise resolving to an array of {@link MailResponse}.
  * @see https://sap.github.io/cloud-sdk/docs/js/features/connectivity/destination#referencing-destinations-by-name
  */
 export async function sendMail<T extends MailConfig>(
   destination: DestinationOrFetchOptions,
   mailConfigs: T | T[],
+  smtpTransportOptions?: SmtpTransportOptions,
```

Comment:
[pp] I am not a big fan of two optional arguments. If you just want to set the last one you need to add a dummy {}. What about using a union type or a larger object with two key?